### DISCUSSION TITLE
display “Error” in recent days

### DISCUSSION THREAD

**Discussion by cgofun** [nhaouari/obsidian-textgenerator-plugin#95] [Use Cases]
![Screenshot_20230212_101928_md obsidian_edit_477168564829793](https://user-images.githubusercontent.com/7732886/218289432-083c3c66-7a23-443e-ba10-e196e402197d.jpg)

![Screenshot_20230212_112524_com microsoft rdc android_edit_484388633258](https://user-images.githubusercontent.com/7732886/218291281-ed677704-86ca-4201-acc1-2dd1ecd0a484.jpg)

Same on PC and mobile phone. How to solve it？

**Comment by LangyMD** [comment]
I'm having the same issue as you - the plugin simply stopped working recently. EDIT: Turns out it's because the trial tokens expired. Woops!

**Comment by dragontlg** [comment]
textgenerator试用令牌已过期如何恢复？
